docs/tutorials/podman_tutorial_cn.md

```diff
@@ -0,0 +1,238 @@
+> - 译文出自：[掘金翻译计划](https://juejin.cn/translate)
+
+
+
+![PODMAN logo](https://github.com/containers/podman/raw/main/logo/podman-logo-source.svg)
+
+Podman是作为libpod库的一部分提供的程序，可以被用于创建和管理容器。
+
+下面的教程会教你如何启动 Podman 并使用 Podman 执行一些基本的命令。
+
+
+如果你正在使用 Mac 或者 Windows ，你应该先查看[Mac和Windows使用说明](https://github.com/containers/podman/blob/master/docs/tutorials/mac_win_client.md)来设置 Podman远程客户端。
+
+
+
+**注意**：这些代码示例旨在作为非root的用户运行，并且在需要root权限的时候使用`sudo`。
+
+
+## 安装Podman
+
+
+
+安装或者构建Podman，请参照[安装说明](https://github.com/containers/podman/blob/master/install.md)。
+
+
+
+## 熟悉podman
+
+
+### 运行一个示例容器
+
+
+这个示例容器会运行一个简单的只有主页的httpd服务器。
+
+```bash
```

Comment:
Please keep this as console as the original doc. This should be a grammar of md docs, don't need change it.